tests/ha/test_ha_pod_restart.py

```diff
@@ -2929,14 +2929,15 @@ def test_continuous_reads_during_pod_restart_ros(self):
             "Step 6: Perform READs and verify DI on the written data in background during "
             "pod restart using %s method", self.restore_method)
         args = {'s3userinfo': list(users.values())[0], 'log_prefix': self.test_prefix,
-                'nclients': 1, 'nsamples': 5, 'skipwrite': True, 'skipcleanup': True,
+                'nclients': 1, 'nsamples': 20, 'skipwrite': True, 'skipcleanup': True,
                 'output': output}
 
         thread = threading.Thread(target=self.ha_obj.event_s3_operation,
                                   args=(event,), kwargs=args)
         thread.daemon = True  # Daemonize thread
         thread.start()
-        time.sleep(HA_CFG["common_params"]["20sec_delay"])
+        # TODO Need to update timing once we get stability in degraded IOs performance
+        time.sleep(200)
```

Comment:
This is for starting some IOs in background before we start other op? and why so much delay, due to perf issue still looks a lot to me.

Reply:
> This is for starting some IOs in background before we start other op? and why so much delay, due to perf issue still looks a lot to me.

Yes @sonalk0209 to start the IOs in background. 
This was still not enough. I observed that 0b is till coming as a part of event set logs. 
![image](https://user-images.githubusercontent.com/83633380/161044770-8baa8a98-8dcc-4219-a5d8-6a5553aa11e4.png)

Reply:
Ok maybe we need to relook once again at this delay